src/Appium.Net/Appium/Android/UiAutomator/AndroidUiScrollable.cs

```diff
@@ -190,11 +184,8 @@ public TerminatedStatementBuilder GetChildByInstance(AndroidUiSelector uiSelecto
         public TerminatedStatementBuilder GetChildByText(AndroidUiSelector uiSelector, string text,
             bool allowScrollSearch = true)
         {
-            if (uiSelector == null)
-                throw new ArgumentNullException(nameof(uiSelector));
-
-            if (text == null)
-                throw new ArgumentNullException(nameof(text));
+            _ = uiSelector ?? throw new ArgumentNullException(nameof(uiSelector));
```

Comment:
this could be further simplified to a method, which looks like

```
object<T> requireNonNull(T o) {
  return o ?? throw new ArgumentNullException(nameof(o))
} 

```

Reply:
Sure, I convert the null guards to a throw-or-access style

Reply:
Take a look at the new iteration on guard clauses and let me know what you think